python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -29,6 +29,28 @@
 
 from paddle.fluid import unique_name
 
+
+class BaseNodeVisitor(gast.NodeVisitor):
```

Comment:
Should be "BaseNodeTransformer" inherited from "gast.NodeTransformer" because Visitor doesn't change node.

Reply:
As discuss offline